tests/platform/cloud/AcceptanceTests_tvm.js

```diff
@@ -0,0 +1,52 @@
+/*
+GPII Integration and Acceptance Testing
+
+Copyright 2014 Fraunhofer IAO
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+ 
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/*global require*/
+
+"use strict";
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii");
+
+gpii.loadTestingSupport();
+
+var testDefs = [
+    {
+        name: "Test the Ticket Vending Machine with Online Flow Manager.",
+        userToken: "tvm_lara",
+        appinfo: encodeURIComponent("{\"OS\":{\"id\":\"web\"},\"solutions\":[{\"id\":\"de.fraunhofer.iao.C4A-TVM\"}]}"),
```

Comment:
''appinfo'' is what I found in the wiki: http://wiki.gpii.net/index.php/Writing_Acceptance_Tests#Applications_using_the_Cloud-Based_Flow_Manager_.28Walkthrough.29. If there is a conflict between the wiki and the other tests, I have no idea which has precedence.